scipy/stats/_continuous_distns.py

```diff
@@ -2075,6 +2075,98 @@ def _entropy(self, c):
 weibull_min = weibull_min_gen(a=0.0, name='weibull_min')
 
 
+class truncweibull_min_gen(rv_continuous):
+    r"""A truncated Weibull minimum continuous random variable.
+
+    %(before_notes)s
+
+    Notes
+    -----
+    The probability density function for `truncweibull_min` is:
+
+    .. math::
+
+        f(x, a, b, c) = \frac{c x^{c-1} \exp(-x^c)}{\exp(-a^c) - \exp(-b^c)}
+
+    for :math:`a < x <= b`, :math:`c > 0`.
+
+    `truncweibull_min` takes :math:`a`, :math:`b`, and :math:`c` as shape
+    parameters.
+
+    Notice that the truncation values, :math:`a` and :math:`b`, is defined in
+    standardized form::
+
+        a = (u_l - loc)/scale
+        b = (u_r - loc)/scale
+
+    where :math:`u_l` and :math:`u_r` are the specific left and right
+    truncation values, respectively.
```

Comment:
This statement could be clarified. What is the support of `truncweibull_min(c, a=1, b=3, loc=4)`: is it `-3 < x < -1`  --- so that in the formulas above `u_l = -3 = a - loc` and `u_e = -1 = b - loc`?

Reply:
Well, from the formulas above you see that `u_l = a*scale + loc` and `u_r = b*scale + loc`. Therefore, the support of `truncweibull_min(c, a=1, b=3, loc=4)` is `5 < x <= 7`. Nonetheless, I'll try to add one or two clarifying sences to the dosctring. Feel free to suggest something.

Reply:
GitHub shows this as outdated, so I'll resolve, but I might add a comment like it to the latest version. This is going to be tough for users, I think. I see why you had to do it this way, though.